setup.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+
+import tahoe_sites
+
+setup(
+    name='tahoe-sites',
+    version=tahoe_sites.__version__,
+    description='Site and organization multi-tenancy management for Appsembler Tahoe',
+    long_description=open('README.md').read(),
+    author='Appsembler',
+    url='https://github.com/appsembler/tahoe-sites',
+    license='MIT License - Copyright (c) 2022 Appsembler',
+    classifiers=[
+        'Framework :: Django',
+        'Framework :: Django :: 2.2',
+        'License :: MIT License - Copyright (c) 2022 Appsembler',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.8',
+    ],
+    packages=find_packages(exclude=['tests']),
+    dependency_links=['edx-organizations @ https://github.com/appsembler/edx-organizations/archive/5.2.0-appsembler13.tar.gz']
```

Comment:
Please add manual dependency via `edx-configs` to simplify future migrations away from `edx-organizations@5.2.0-appsembler13.tar.gz`. 

```suggestion
```